logstash/README.md

```diff
@@ -11,6 +11,10 @@ Get metrics from Logstash service in real time to:
 
 ### Configuration
 
+Logstash is a community-supported integration so it's not packaged with the agent. 
```

Comment:
We should come up with some standard wording here and put it in the main README, as it applies to all integrations-extras. Let's remove it from the PR and I'll add it there.

Reply:
👍